Wireframe/css/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    text-align: center;
+    background: linear-gradient(#D7CCC8, #908f89);
+    color: #FAEBD7;
+    margin: 0;
+}
+
+#logo {
+    width: 10rem;
+    vertical-align: middle;
+}
+
+img {
+    width: 10rem;
+    border-radius: 1rem;
+}
+
+header {
+    color: #780000;
+    background-color: #FAEBD7;
+    background: radial-gradient( #DAD7CD 50%, #908f89);
+    margin: 0 auto;
+    text-shadow:  0 0 1px white;
+    box-shadow: 0 0 15px #780000;
+    padding: 10px;
+}
+
+main {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 3vw auto;
+}
+
+h3 {
+    text-shadow: 0 0 5px black;
+}
+
+article {
+    border: 4px solid #780000;
+    border-radius: 30px;
+    padding: 2rem;
+    text-shadow: 0 0 2px black;
+    box-shadow: 0 0 10px black;
+}
+
+article:first-child {
+    background: radial-gradient(#34779b, #003049);
+    width: 90vw;
+    margin: 3vw auto 2vw;
+}
+
+#second, #third {
+    width: 40vw;
+    margin: 3vw auto;
+    background: radial-gradient( #003049, #34779b);
+}
```

Comment:
is there a way to apply these styles to the second and third articles without using IDs?
(hint: they are NOT the first article -- how does cascading help?)

Reply:
I’ll update the main element’s style so that only the first article is styled differently using the :first-child selector. The cascading styles will then apply to the second and third articles by default without needing IDs.
Thanks for mentioning it!